user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md

```diff
@@ -41,6 +41,7 @@ A `FieldDescriptor` object defines what a field of a `DomInstance` should look l
 >
 > - When you store `DateTime` values, you can save them in either the local or UTC time zone. However, we strongly recommend always using UTC. When a `DateTime` field value is displayed in a low-code app form, the values will be converted to the time zone set by the browser. When it is updated via the form, the value will be converted and saved in UTC even if the value was originally stored in local time.
 > - When a `string` value is stored, from DataMiner 10.4.12/10.5.0 onwards, the value is limited to a maximum of 32 766 UTF-8 bytes, so the maximum number of characters will have to be lower.<!-- RN 39496 -->
+> - When a `TimeSpan` value is used, make sure its value does not exceed (+ or -) 10,000 years.
```

Comment:
To avoid possible confusion, I'd prefer to leave out the "(+ or -)".

```suggestion
> - When a `TimeSpan` value is used, make sure its value does not exceed 10,000 years.
```

Reply:
Ok for me. It was intended to make it clear that it cannot be less than -10,000 years as well. But maybe not needed?

Reply:
Will a negative value ever be used here? If yes, how about this alternative? If no, indeed we'd better not mention this.
```suggestion
> - When a `TimeSpan` value is used, make sure its value does not exceed 10,000 or go below -10,000 years.
```

Reply:
Yes, it is possible that users may assign negative timespans here. I have committed the suggestion. Thanks!